### PR TITLE
Add Gradle wrapper support

### PR DESCRIPTION
**Pull request by Theosakamg**
Colcon can now launch local project wrapper.

### CODE REVIEW

File: colcon_gradle/task/gradle/__init__.py [hunk 1]
```diff
@@ -12,6 +12,8 @@
 GRADLE_COMMAND_ENVIRONMENT_VARIABLE = EnvironmentVariable(
     'GRADLE_COMMAND', 'The full path to the Gradle executable')
 
+"""Check OS"""
+IS_WINDOWS = os.name == 'nt' # TODO need to be in Colcon-core ?
```
Comment (esteve) [inline colcon_gradle/task/gradle/__init__.py]: Please remove the comment.

File: colcon_gradle/task/gradle/build.py [hunk 2]
```diff
@@ -62,13 +63,34 @@ def add_arguments(self, *, parser):  # noqa: D102
...
+
+        # Gradle Arguments
+        cmd += args.gradle_args
+
+        # invoke build step
         return await check_call(
             self.context, cmd, cwd=args.build_base, env=env)
+
+def has_local_executable(args):
+    gradle_path   = get_local_executable(args)
```
Comment (esteve) [inline colcon_gradle/task/gradle/build.py]: `gradle_path = get_local_executable(args)`

File: colcon_gradle/task/gradle/build.py [hunk 3]
```diff
@@ -62,13 +63,34 @@ def add_arguments(self, *, parser):  # noqa: D102
...
+    return gradle_path.is_file()
+
+def get_local_executable(args):
+    gradle_script = 'gradlew.bat' if IS_WINDOWS else 'gradlew'
+    gradle_path   = Path(args.path) / gradle_script
```
Comment (esteve) [inline colcon_gradle/task/gradle/build.py]: `gradle_path = Path(args.path) / gradle_script`

**Comment by Theosakamg** [comment]
Done

**Comment by esteve** [comment]
Thanks! I'll squash the commits and merge it.

**Pull request merged**